embedded-workshop-book/src/preparations.md

```diff
@@ -0,0 +1,31 @@
+# Preparations
+
+This chapter contains informations about the course material, the required hardware and an installation guide. 
+
+
+## Required Hardware 
+
+- [nRF52840 Development Kit (DK)](https://www.nordicsemi.com/Software-and-Tools/Development-Kits/nRF52840-DK)
+- [nRF52840 Dongle](https://www.nordicsemi.com/Software-and-tools/Development-Kits/nRF52840-Dongle)
+- 2 micro-USB cables
+- 2 available USB-A ports on your laptop / PC (you can use a USB hub if you don't have enough ports)
+
+In this workshop we'll use both the nRF52840 Development Kit (DK) and the nRF52840 Dongle. We'll mainly develop programs for the DK and use the Dongle to assist with some exercises.
+
+For the span of this workshop keep the nRF52840 DK connected to your PC using a micro-USB cable. Connect the USB cable to the J2 port on the nRF52840 DK. Instructions to identify the USB ports on the nRF52840 board can be found in the top level README file.
+
+
+## Required Software
+Please install the required software before the course starts. 
+
+## Course Material
+This book contains the material for the beginner and the advanced workshop in embedded Rust. It aims to be as inclusive as possible. This means, that some information is available in several forms, for example pictures and text description. We also use icons so that different kinds of information are visually distiguishable on the first glance. If you have accessibility needs that are not covered, please let us know.
+
+# Icons and Formatting we use
+We use Icons to mark different kinds of information in the book:
```

Comment:
✨ love this idea. I'm not sure if I'm mistaken (still haven't learned to operate a screen reader to test it out) nut– Accessibility nitpick for the future:
🔎 is called "Left-Pointing Magnifying Glass" which could be a bit annoying but then again teh only alternative I could come up with is 🔦 "Electric Torch" which is not as nice and also not much more concise.